resources/code_of_conduct.Rmd

```diff
@@ -0,0 +1,123 @@
+
+The DaSEH community is committed to ethical partnerships and avoiding partnerships with entities whose practices are inconsistent with DaSEH values. We encourage community members to report or raise concerns about partnerships they deem unethical to the instructors or via the reporting form.
+
+#### Examples of unacceptable harassment, intimidation, and bullying behavior
+
+Harassment includes, but is not limited to:
+
+- Making comments in chats, to an audience or personally, that belittle or demean another person  
+- Sharing sexual images online  
+- Harassing photography or recording  
+- Sustained disruption of talks or other events  
+- Unwelcome sexual attention  
+- Advocating for, or encouraging, any of the above behavior  
+
+Intimidation and bullying include, but are not limited to:
+
+- Aggressive or browbeating behavior  
+- Mocking or insulting another person’s intellect, work, perspective, or question/comment  
+- Making reference to someone’s gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or other personal attributes in the context of a scientific discussion  
+- Deliberately making someone feel unwelcome  
+- Trolling behavior (deliberately inflammatory or offensive messages)  
+- Sustained off-topic posts  
+
+## Enforcement
+
+Anyone asked to stop harassing or intimidating behavior are expected to comply immediately.
+
+If a person contravenes the Code of Conduct, the DaSEH organizers retain the right to take any action that ensures a welcoming environment for all community members. This includes warning the alleged offender or temporary/permanent expulsion from the event and/or electronic platforms under DaSEH’s control.
+
+The organizers may take action to redress anything designed to, or with the clear impact of, disrupting an event or electronic communication platform or making the environment hostile for any community member.
+
+We expect everyone in the DaSEH community to comply with the Code of Conduct when participating in events and online communication platforms.
+
+## Reporting
+
+If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. You can make a report either anonymously or personally. All reports will be reviewed by the organizers and will be kept confidential.
+
+#### Electronically
+
+You can make an anonymous or non-anonymous report via the following link: https://forms.gle/WBymAnzfWYpfWk4b6. It is a free-form text box that will be forwarded to the organizers. If you are uncomfortable reporting to the organizers as a group, you can contact any individual instructor via email or a direct message on the Slack workspace (https://daseh.slack.com/). Please include screenshots/copies of all relevant electronic conversations whenever possible (you don’t need to compromise your anonymity!).
```

Comment:
I really like this form. I feel like we could use this for the ITN slack. Do you mind sharing it with me? (putting it in the shared drive might work??)